scout/server/blueprints/variants/views.py

```diff
@@ -54,7 +54,7 @@ def variants(institute_id, case_name):
     user_obj = store.user(current_user.email)
 
     if request.method == "POST":
-        # If special filter buttons were selected:
+        # If special filter were selected:
```

Comment:
"special"? Simply if filter was selected perhaps? Is that really the only reason for POST nowadays?

Reply:
There are no such comments elsewhere describing what entry points may have got there. Can you see any reason why we should not better just remove the comment?

Reply:
You are right, I'll remove it